be/src/vec/exec/join/vnested_loop_join_node.cpp

```diff
@@ -544,7 +545,8 @@ Status VNestedLoopJoinNode::_do_filtering_and_update_visited_flags(Block* block,
 
             const size_t size = filter.size();
             for (size_t i = 0; i < size; ++i) {
-                filter_data[i] &= !null_map[i];
+                filter_data[i] = filter_null ? (filter_data[i] || null_map[i])
```

Comment:
the origin code look can simd，make sure the new code is better？
maybe the better way is
```
if(filter_null) {
   for ...
} else {
   for...
}
```